scss/main.scss

```diff
@@ -102,3 +102,5 @@ nav {
     }
   }
 }
+
+@import 'responsive';
```

Comment:
@jehna and I don't think you're right about overriding... we're using `@media`, the browser will understand that it has higher priority